requirements/static/py3.5/linux.txt

```diff
@@ -10,9 +10,7 @@ asn1crypto==0.24.0        # via cryptography
 atomicwrites==1.3.0       # via pytest
 attrs==19.1.0             # via pytest
 aws-xray-sdk==0.95        # via moto
-backports-abc==0.5        # via tornado
 backports.functools-lru-cache==1.5  # via cheroot
-backports.ssl-match-hostname==3.7.0.1  # via websocket-client
```

Comment:
@s0undt3ch is this normal for pre-commit to remove this? seems like we would want this backports. maybe i ran something wrong?

Reply:
It's the second time these changes happen.
Can you please try `pre-commit clean` and run it again?
I know you're on Manjaro, and you probably installed to the system python, that means 3.8.
Can you try the above and confirm the other questions please?

Reply:
i think i was actually using my python2 virtualenv, im running into errors on 3.8 though, what version of python are you running it successfully on?

Reply:
3.5 or 3.6, can't remember.